app/schemas.py

```diff
@@ -242,14 +242,7 @@ def _get_allowed_broadcast_provider(self, service):
         return service.allowed_broadcast_provider
 
     def _get_broadcast_channel(self, service):
```

Comment:
Do we need the method anymore? It looks like it's only used once, just above.

Reply:
I think so, or at least I'm just copying what we are doing for `_get_allowed_broadcast_provider` so I assume that one was needed too (it's only used above as well).

Reply:
Oh yes, good point. I did a bit of quick reading marshmallow, and this does seem to be the technique to use if we want to serialize an association without defining a nested schema for it.

Note to self: these schemas are used for a combination of API responses ([here](https://github.com/alphagov/notifications-api/blob/1b22de3830181809fb790cb06b9c2b647ce01cb3/app/service/rest.py#L191)), for which this attribute is required; and reduced DB queries ([here](https://github.com/alphagov/notifications-api/blob/1b22de3830181809fb790cb06b9c2b647ce01cb3/app/celery/tasks.py#L194)), for which it's not necessary (`dump_only`).